jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/HttpSender.java

```diff
@@ -66,6 +77,7 @@ public void send(Process process, List<Span> spans) throws SenderException {
     try {
       response = httpClient.newCall(request).execute();
     } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
don't we have that elsewhere?

Reply:
That was my mistake not to remove debug code. I will delete.